.github/scripts/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM fedora:34
```

Comment:
Why not CentOS Stream 8? That seems to me more production-like choice.

Reply:
Agreed. I will try it out and evaluate.

Reply:
Resulting image when using CentOS 8 Stream had ~ 800MB while with Fedora 34 we got ~ 450MB.
@kgiusti @ganeshmurthy any thoughts?

Reply:
I was looking here - https://github.com/interconnectedcloud/qdr-image/blob/master/Dockerfile and that Dockerfile is using Fedora. In the past, was the image created from that Dockerfile ?

Reply:
That is correct Ganesh. The former image was using Fedora 30. Some deps like libnghttp2 need at least Fedora 33
to be used. But since we now require CMake 3.20, I had to bump it to Fedora 34.

Reply:
I meant that since the former image was using Fedora, there is no harm in the current image using Fedora as well. Fedora 34 sounds good.